src/port1.0/portutil.tcl

```diff
@@ -508,6 +508,16 @@ proc default_check {optionName index op} {
     }
 }
 
+# env_default 
+# Unless port is Xcode dependent, returns env which sets developer_dir to Command Line Tools
+proc env_default {} {
+    global use_xcode
+    if {[info exists use_xcode] && $use_xcode eq "yes"} {
```

Comment:
I don't think you need the `[info exists use_xcode]` here, since it's already defined as an option.

Reply:
Removed